app/views/bookings/new.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <%= simple_form_for([@kondo, @booking]) do |f| %>
-    <%= f.input :booked_date, as: :datetime, html5: true %>
+    <%= f.datetime_field :booked_date, min: Date.today, as: :date, html5: true %>
```

Comment:
In reality I don't think same day booking would be possible but...

So I tried to maybe make the bookings 3 days ahead of time however this ruins the UI a little, no more "Today".
```ruby
min: Date.today + 3
```
<img width="418" alt="Screen Shot 2022-01-25 at 18 39 02" src="https://user-images.githubusercontent.com/18746187/150951379-cb759ff6-9182-43ab-8caa-af0e52ee4b0d.png">